### PR TITLE
Optimization of volumemanager: processing non-pvc volumes preferentially to ensure system critical pods do not blocked after kubelet restarted.

### PR DESCRIPTION
**Pull request by sanwishe**
Signed-off-by: 姜明志10170261 <jiang.mingzhi35@zte.com.cn>

**What type of PR is this?**
/kind flake

**What this PR does / why we need it**:

In some cases when kubelet restarted, non PVC volumes might be blocked adding to dsw cache since apiserver is not ready for fetching PVC information. Processing non PVC volumes in preference to PVC volumes would work for this situation

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
/kind bug
/sig storage

Can you please add tests for this?

**Comment by sanwishe** [comment]
@SergeyKanzhelev Ok, I'll add test for this.

**Comment by sanwishe** [comment]
I signed it

**Comment by thockin** [comment]
Hi @jingxu97 and @gnufied and/or @msau42 

This has been stalled and got assigned to me.  I am clearly not the right person to review it.  Can one of you please take a look?

**Comment by jingxu97** [comment]
I missed this PR. I will take a look at it today.

File: pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go [hunk 1]
```diff
@@ -142,6 +142,10 @@ func (dswp *desiredStateOfWorldPopulator) Run(sourcesReady config.SourcesReady,
 		dswp.populatorLoop()
 		return done, nil
 	}, stopCh)
+
+	// Call dswp.populatorLoop here to process non PVC volumes first of all
+	dswp.populatorLoop()
```
Comment (jingxu97) [inline pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go:147]: populatorLoop is already called in line 142?

**Comment by jingxu97** [comment]
@sanwishe could you provide some details about how non-pvc volumes is blocked?

**Comment by sanwishe** [comment]
> @sanwishe could you provide some details about how non-pvc volumes is blocked?

while upgrading a cluster, the upgrade processes as the folowing steps:

- replace the manifists of control-plane and static loadbalancer pods
- restart kubelet

after kubelet restarted, it kills the pods defined by this manifests and rebuild them. A small probability status is: kube-apiserver and loadbalancer were killed just after kubelet has synchronized all business pods from kube-apiserver.  In the main loop of `desiredStateOfWorldPopulator.populatorLoop()`, it can be take almost 3s to connect kube-apiserver to fetch each pvc detail when loadbalancer's vip is not up.

```
func (dswp *desiredStateOfWorldPopulator) getPVCExtractPV(
	namespace string, claimName string) (*v1.PersistentVolumeClaim, error) {
	pvc, err :=
		dswp.kubeClient.CoreV1().PersistentVolumeClaims(namespace).Get(context.TODO(), claimName, metav1.GetOptions{})
	if err != nil || pvc == nil {
		return nil, fmt.Errorf("failed to fetch PVC from API server: %v", err)
	}
```

 In our cluster, there are more then 50 pvcs per node, it sometimes takes more than 2mintes to add all volumes to desiredStateOfWorld. the system critical pods may appear back in order in the iteration of `dswp.podManager.GetPods()`.

As described by the kubelet's log, each error of fecthing pvc from kube-apiserver takes more than 3s.
```
I0702 02:21:12.462550   22155 desired_state_of_world_populator.go:130] Desired state populator starts to run
E0702 02:22:41.869803   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "mo-ne-svr-momodel-ms-2-6snts_ranoss(7d07d39b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:11.870266   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmdata" for pod "cm-tps-0519-tpworker-ms-1-gc45v_ranoss(8fe8cdcd-b538-11ea-9b5e-48df37339c6c)": error processing PVC "ranoss"/"home-cmdata": failed to fetch PVC ranoss/home-cmdata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmdata: dial tcp 193.101.0.38:7443: i/o timeout
E0702 02:23:19.945647   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "cm-tps-0519-tpworker-ms-1-gc45v_ranoss(8fe8cdcd-b538-11ea-9b5e-48df37339c6c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:28.963583   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "dcp-event-subscription-dcp-event-subscription-ms-3-vrvfl_ranoss(c7ff9248-ba89-11ea-b466-48df371cc760)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:31.969577   22155 desired_state_of_world_populator.go:296] Error processing volume "data-gr" for pod "dcp-event-subscription-dcp-event-subscription-ms-3-vrvfl_ranoss(c7ff9248-ba89-11ea-b466-48df371cc760)": error processing PVC "ranoss"/"data-gr": failed to fetch PVC ranoss/data-gr from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/data-gr: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:34.975830   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp" for pod "dcp-event-subscription-dcp-event-subscription-ms-3-vrvfl_ranoss(c7ff9248-ba89-11ea-b466-48df371cc760)": error processing PVC "ranoss"/"home-dcp": failed to fetch PVC ranoss/home-dcp from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:37.981589   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcm-data" for pod "dcp-event-subscription-dcp-event-subscription-ms-3-vrvfl_ranoss(c7ff9248-ba89-11ea-b466-48df371cc760)": error processing PVC "ranoss"/"home-dcm-data": failed to fetch PVC ranoss/home-dcm-data from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcm-data: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:40.987765   22155 desired_state_of_world_populator.go:296] Error processing volume "jstorm-topo" for pod "stream-common-supervisor-ms-2-7j7zl_ranoss(0f8bdab4-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"jstorm-topo": failed to fetch PVC ranoss/jstorm-topo from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/jstorm-topo: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:43.993619   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmdata" for pod "stream-common-supervisor-ms-2-7j7zl_ranoss(0f8bdab4-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-cmdata": failed to fetch PVC ranoss/home-cmdata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmdata: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:46.999777   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "stream-common-supervisor-ms-2-7j7zl_ranoss(0f8bdab4-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:50.005682   22155 desired_state_of_world_populator.go:296] Error processing volume "pmfiles" for pod "stream-common-supervisor-ms-2-7j7zl_ranoss(0f8bdab4-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"pmfiles": failed to fetch PVC ranoss/pmfiles from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/pmfiles: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:53.011591   22155 desired_state_of_world_populator.go:296] Error processing volume "naf-rsync" for pod "stream-common-supervisor-ms-2-7j7zl_ranoss(0f8bdab4-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"naf-rsync": failed to fetch PVC ranoss/naf-rsync from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/naf-rsync: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:56.017611   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp-rnc-model" for pod "stream-common-supervisor-ms-2-7j7zl_ranoss(0f8bdab4-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp-rnc-model": failed to fetch PVC ranoss/home-dcp-rnc-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp-rnc-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:23:59.023648   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp" for pod "stream-common-supervisor-ms-2-7j7zl_ranoss(0f8bdab4-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp": failed to fetch PVC ranoss/home-dcp from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:02.029628   22155 desired_state_of_world_populator.go:296] Error processing volume "jstorm-topo" for pod "stream-common-supervisor-ms-2-fbk4d_ranoss(0fd0844b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"jstorm-topo": failed to fetch PVC ranoss/jstorm-topo from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/jstorm-topo: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:05.035696   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmdata" for pod "stream-common-supervisor-ms-2-fbk4d_ranoss(0fd0844b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-cmdata": failed to fetch PVC ranoss/home-cmdata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmdata: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:08.041706   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "stream-common-supervisor-ms-2-fbk4d_ranoss(0fd0844b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:11.047615   22155 desired_state_of_world_populator.go:296] Error processing volume "pmfiles" for pod "stream-common-supervisor-ms-2-fbk4d_ranoss(0fd0844b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"pmfiles": failed to fetch PVC ranoss/pmfiles from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/pmfiles: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:14.053607   22155 desired_state_of_world_populator.go:296] Error processing volume "naf-rsync" for pod "stream-common-supervisor-ms-2-fbk4d_ranoss(0fd0844b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"naf-rsync": failed to fetch PVC ranoss/naf-rsync from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/naf-rsync: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:20.065811   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp-rnc-model" for pod "stream-common-supervisor-ms-2-fbk4d_ranoss(0fd0844b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp-rnc-model": failed to fetch PVC ranoss/home-dcp-rnc-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp-rnc-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:26.077696   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp" for pod "stream-common-supervisor-ms-2-fbk4d_ranoss(0fd0844b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp": failed to fetch PVC ranoss/home-dcp from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:29.083748   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmdata" for pod "mo-ne-svr-mohandle-ms-2-9n6lj_ranoss(7d09d516-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-cmdata": failed to fetch PVC ranoss/home-cmdata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmdata: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:32.089670   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "mo-ne-svr-mohandle-ms-2-9n6lj_ranoss(7d09d516-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:35.095709   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp-ca" for pod "dcp-collection-agent-dcp-collection-agent-ms-1-trlvr_ranoss(671975aa-ab92-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp-ca": failed to fetch PVC ranoss/home-dcp-ca from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp-ca: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:38.101706   22155 desired_state_of_world_populator.go:296] Error processing volume "jstorm-topo" for pod "stream-common-supervisor-ms-2-nkww6_ranoss(107862b7-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"jstorm-topo": failed to fetch PVC ranoss/jstorm-topo from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/jstorm-topo: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:41.107689   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmdata" for pod "stream-common-supervisor-ms-2-nkww6_ranoss(107862b7-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-cmdata": failed to fetch PVC ranoss/home-cmdata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmdata: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:44.113787   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "stream-common-supervisor-ms-2-nkww6_ranoss(107862b7-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:47.119674   22155 desired_state_of_world_populator.go:296] Error processing volume "pmfiles" for pod "stream-common-supervisor-ms-2-nkww6_ranoss(107862b7-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"pmfiles": failed to fetch PVC ranoss/pmfiles from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/pmfiles: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:50.125752   22155 desired_state_of_world_populator.go:296] Error processing volume "naf-rsync" for pod "stream-common-supervisor-ms-2-nkww6_ranoss(107862b7-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"naf-rsync": failed to fetch PVC ranoss/naf-rsync from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/naf-rsync: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:53.131766   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp-rnc-model" for pod "stream-common-supervisor-ms-2-nkww6_ranoss(107862b7-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp-rnc-model": failed to fetch PVC ranoss/home-dcp-rnc-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp-rnc-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:56.137568   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp" for pod "stream-common-supervisor-ms-2-nkww6_ranoss(107862b7-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp": failed to fetch PVC ranoss/home-dcp from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:24:59.143649   22155 desired_state_of_world_populator.go:296] Error processing volume "sal-kafka-data" for pod "netconfadapter-netconfadapter-ms-3-cqcw2_ranoss(823d3f95-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"sal-kafka-data": failed to fetch PVC ranoss/sal-kafka-data from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/sal-kafka-data: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:02.149753   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmconsistency" for pod "cm-data-check-worker-ms-4-vw6nj_ranoss(c1e0758c-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-cmconsistency": failed to fetch PVC ranoss/home-cmconsistency from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmconsistency: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:05.155574   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "cm-data-check-worker-ms-4-vw6nj_ranoss(c1e0758c-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:08.161316   22155 desired_state_of_world_populator.go:296] Error processing volume "pmfiles" for pod "pm-collection-ran-pm-datfileupload-ranomm-ms-3-vg7vp_ranoss(fe091877-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"pmfiles": failed to fetch PVC ranoss/pmfiles from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/pmfiles: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:10.165508   22155 desired_state_of_world_populator.go:296] Error processing volume "pmlogs" for pod "pm-collection-ran-pm-datfileupload-ranomm-ms-3-vg7vp_ranoss(fe091877-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"pmlogs": failed to fetch PVC ranoss/pmlogs from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/pmlogs: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:13.171533   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model-sdrmoapt" for pod "u31service-sdrmoapt-ms-2-s4rgm_ranoss(7e7669fc-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:21.187771   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "rnccmservice-single-instance-pod-2-54k84_ranoss(c22ecb76-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:24.193650   22155 desired_state_of_world_populator.go:296] Error processing volume "home-u31ftp" for pod "rnccmservice-single-instance-pod-2-54k84_ranoss(c22ecb76-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-u31ftp": failed to fetch PVC ranoss/home-u31ftp from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-u31ftp: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:27.199792   22155 desired_state_of_world_populator.go:296] Error processing volume "home-swr" for pod "sus-base-sus-workflow-ms-4-2cpfx_ranoss(0d7d6979-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-swr": failed to fetch PVC ranoss/home-swr from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-swr: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:30.205675   22155 desired_state_of_world_populator.go:296] Error processing volume "home-sus" for pod "sus-base-sus-workflow-ms-4-2cpfx_ranoss(0d7d6979-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-sus": failed to fetch PVC ranoss/home-sus from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-sus: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:33.211734   22155 desired_state_of_world_populator.go:296] Error processing volume "home-upgradedata" for pod "sus-base-sus-workflow-ms-4-2cpfx_ranoss(0d7d6979-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-upgradedata": failed to fetch PVC ranoss/home-upgradedata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-upgradedata: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:36.218260   22155 desired_state_of_world_populator.go:296] Error processing volume "jstorm-topo" for pod "stream-common-supervisor-ms-2-t745h_ranoss(0f0a329e-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"jstorm-topo": failed to fetch PVC ranoss/jstorm-topo from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/jstorm-topo: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:39.223616   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmdata" for pod "stream-common-supervisor-ms-2-t745h_ranoss(0f0a329e-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-cmdata": failed to fetch PVC ranoss/home-cmdata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmdata: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:42.229694   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "stream-common-supervisor-ms-2-t745h_ranoss(0f0a329e-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:45.235762   22155 desired_state_of_world_populator.go:296] Error processing volume "pmfiles" for pod "stream-common-supervisor-ms-2-t745h_ranoss(0f0a329e-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"pmfiles": failed to fetch PVC ranoss/pmfiles from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/pmfiles: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:25:48.241723   22155 desired_state_of_world_populator.go:296] Error processing volume "naf-rsync" for pod "stream-common-supervisor-ms-2-t745h_ranoss(0f0a329e-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"naf-rsync": failed to fetch PVC ranoss/naf-rsync from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/naf-rsync: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:06.277770   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp-rnc-model" for pod "stream-common-supervisor-ms-2-t745h_ranoss(0f0a329e-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp-rnc-model": failed to fetch PVC ranoss/home-dcp-rnc-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp-rnc-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:09.283650   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp" for pod "stream-common-supervisor-ms-2-t745h_ranoss(0f0a329e-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp": failed to fetch PVC ranoss/home-dcp from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:12.289648   22155 desired_state_of_world_populator.go:296] Error processing volume "jstorm-topo" for pod "stream-common-supervisor-ms-2-zh6sf_ranoss(10adc24a-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"jstorm-topo": failed to fetch PVC ranoss/jstorm-topo from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/jstorm-topo: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:15.295900   22155 desired_state_of_world_populator.go:296] Error processing volume "home-cmdata" for pod "stream-common-supervisor-ms-2-zh6sf_ranoss(10adc24a-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-cmdata": failed to fetch PVC ranoss/home-cmdata from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-cmdata: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:18.301778   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "stream-common-supervisor-ms-2-zh6sf_ranoss(10adc24a-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:24.313739   22155 desired_state_of_world_populator.go:296] Error processing volume "pmfiles" for pod "stream-common-supervisor-ms-2-zh6sf_ranoss(10adc24a-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"pmfiles": failed to fetch PVC ranoss/pmfiles from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/pmfiles: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:27.319940   22155 desired_state_of_world_populator.go:296] Error processing volume "naf-rsync" for pod "stream-common-supervisor-ms-2-zh6sf_ranoss(10adc24a-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"naf-rsync": failed to fetch PVC ranoss/naf-rsync from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/naf-rsync: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:30.325632   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp-rnc-model" for pod "stream-common-supervisor-ms-2-zh6sf_ranoss(10adc24a-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp-rnc-model": failed to fetch PVC ranoss/home-dcp-rnc-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp-rnc-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:33.331701   22155 desired_state_of_world_populator.go:296] Error processing volume "home-dcp" for pod "stream-common-supervisor-ms-2-zh6sf_ranoss(10adc24a-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-dcp": failed to fetch PVC ranoss/home-dcp from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-dcp: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:36.337583   22155 desired_state_of_world_populator.go:296] Error processing volume "home-model" for pod "mo-ne-svr-momodel-ms-2-6snts_ranoss(7d07d39b-ab91-11ea-b73c-48df371cc75c)": error processing PVC "ranoss"/"home-model": failed to fetch PVC ranoss/home-model from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-model: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:39.343747   22155 desired_state_of_world_populator.go:296] Error processing volume "home-plugins" for pod "pluginmanager-pluginmng-ms-2-tmjjh_ranoss(3ca7a2e1-9860-11ea-900a-48df37339c6c)": error processing PVC "ranoss"/"home-plugins": failed to fetch PVC ranoss/home-plugins from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/home-plugins: dial tcp 193.101.0.38:7443: connect: no route to host
E0702 02:26:42.351555   22155 desired_state_of_world_populator.go:296] Error processing volume "data-gr" for pod "sm-sm-backup-2-p2kbq_ranoss(3e947404-ab72-11ea-8d7b-48df371cc75c)": error processing PVC "ranoss"/"data-gr": failed to fetch PVC ranoss/data-gr from API server. err=Get https://193.101.0.38:7443/api/v1/namespaces/ranoss/persistentvolumeclaims/data-gr: dial tcp 193.101.0.38:7443: connect: no route to host

```

File: pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go (hunk 1 above)
Comment (sanwishe) [reply pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go:147]: yes, it is duplicate.

**Pull request closed without merging**